app/src/main/java/com/x8bit/bitwarden/data/tools/generator/repository/GeneratorRepositoryImpl.kt

```diff
@@ -204,73 +201,6 @@ class GeneratorRepositoryImpl(
                 },
             )
 
-    @Suppress("LongMethod", "CyclomaticComplexMethod")
-    override fun getPasswordGeneratorPolicy(): PolicyInformation.PasswordGenerator? {
```

Comment:
So this was never used?

Reply:
it was removed on [this](https://github.com/bitwarden/android/pull/3708/commits/5bcc50683809b07cc9ad477ce0cb4423eaf6eb3a#diff-90a28721ac9c1302004c349dbe415eb619dae82eac3713cce169312c35a88b5aL287-L291) commit https://github.com/bitwarden/android/pull/3708/commits/5bcc50683809b07cc9ad477ce0cb4423eaf6eb3a , 
we were using the method to return the `PasscodeType`, and that can be obtained from the policy itself, seemed to me that `getPasswordGeneratorPolicy` was too complex for that purpose